### PR TITLE
Add Marionette updates for release 99

### PR DESCRIPTION
**Pull request by juliandescottes**
#### Summary
Add releases notes for marionette

#### Supporting details
Bug list https://bugzilla.mozilla.org/buglist.cgi?j_top=OR&o1=equals&resolution=FIXED&o2=equals&query_format=advanced&v1=fixed&component=Marionette&v2=verified&product=Testing&f1=cf_status_firefox99&list_id=16043355

#### Related issues
From devtools release tasks https://bugzilla.mozilla.org/show_bug.cgi?id=1758471

#### Metadata
- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by juliandescottes** [comment]
cc @whimboo

File: files/en-us/mozilla/firefox/releases/99/index.md [hunk 1]
```diff
@@ -42,6 +42,10 @@ No notable changes.
   ({{bug(1720353)}}).
 
 
+### WebDriver conformance (Marionette)
+
+- Fixed a bug where the Shift key was not handled properly when part of a sequence ({{bug(1757636)}}).
```
Comment (whimboo) [inline files/en-us/mozilla/firefox/releases/99/index.md:47]: Just one nit to make it clear about which sequence we are talking about here:

```suggestion
- Fixed a bug where the Shift key was not handled properly when part of a key sequence of the <code>WebDriver:ElementSendKeys</code> command ({{bug(1757636)}}).
```

**Comment by juliandescottes** [comment]
cc @Rumyra for review

File: files/en-us/mozilla/firefox/releases/99/index.md [hunk 2]
```diff
@@ -42,6 +42,10 @@ No notable changes.
   ({{bug(1720353)}}).
 
 
+### WebDriver conformance (Marionette)
+
+- Fixed a bug where the Shift key was not handled properly when part of a key sequence of the <code>WebDriver:ElementSendKeys</code> command ({{bug(1757636)}}).
```
Comment (whimboo) [inline files/en-us/mozilla/firefox/releases/99/index.md:47]: Sorry, but I was actually wrong. The `<code>` tag isn't used anymore but instead we make use of back ticks for the commands:

```suggestion
- Fixed a bug where the Shift key was not handled properly when part of a key sequence of the `WebDriver:ElementSendKeys` command ({{bug(1757636)}}).
```

**Comment by Rumyra** [comment]
Thanks both!

**Review by Rumyra — APPROVED** [review]

**Comment by juliandescottes** [comment]
Thanks!

**Pull request merged**